common/common.h

```diff
@@ -66,6 +66,7 @@ struct gpt_params {
     int32_t grp_attn_w                      = 512;   // group-attention width
     float   rope_freq_base                  = 0.0f;  // RoPE base frequency
     float   rope_freq_scale                 = 0.0f;  // RoPE frequency scaling factor
+    int32_t token_interval                  = 512;   // show token count every 512 tokens
```

Comment:
Minor : move this after `grp_attn_w`. The idea is to not split the rope related parameters (rope, yarn) from each other